Components/alert/alert.css

```diff
@@ -17,47 +17,47 @@
     margin: 2rem 0;
 }
 
-.primary-alert{
+.primary-alert {
     width: 100%;
     padding: 1rem 3rem 1rem 1rem;
     margin: 1rem 0;
     font-size: 1rem;
-    background-color:  var(--primary-color);
+    background-color: var(--primary-color);
     border-radius: 7px;
 }
 
-.secondary-alert{
+.secondary-alert {
     width: 100%;
     padding: 1rem 3rem 1rem 1rem;
     margin: 1rem 0;
     font-size: 1rem;
-    background-color:  var(--secondary-color);
+    background-color: var(--secondary-color);
     border-radius: 7px;
 }
 
-.success-alert{
+.success-alert {
     width: 100%;
     padding: 1rem 3rem 1rem 1rem;
     margin: 1rem 0;
     font-size: 1rem;
```

Comment:
All alerts have the same width, padding, margin, font size, and border radius. You can create a single class called alert which will contain all these common properties and use that class along with other specific classes for each alert.
For e.g 
``` 
<div class="alert success-alert"> </div>
```